themes/default/content/registry/packages/awsx/_index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: AWSx
+meta_desc: Well-Architected Infrastructure as Code for AWS. The easiest way to AWS — from development to production.
+layout: overview
+---
+
+The Amazon Web Services (AWS) Crosswalk (AWSx) provider for Pulumi can provision many of the cloud resources available in [AWS](https://aws.amazon.com/). It uses the AWS SDK to manage and provision resources.
+
+The AWSx provider must be configured with credentials to deploy and update resources in AWS; see [Installation & Configuration]({{<relref "./installation-configuration">}}) for instructions.
+
+**New to Pulumi and AWS?** [Get started with AWS using our AWSx tutorial]({{<relref "docs/guides/crosswalk/aws/">}}).
```

Comment:
I’m thumbs, but can take a closer look at this PR in an hour or so. Noticed this though that’ll need to be fixed:

```suggestion
**New to Pulumi and AWS?** [Get started with AWS using our AWSx tutorial]({{<relref "/docs/guides/crosswalk/aws">}}).
```